tests/lms/views/decorators/test_h_api.py

```diff
@@ -7,7 +7,11 @@
 from pyramid.httpexceptions import HTTPBadGateway
 from pyramid.httpexceptions import HTTPBadRequest
 from pyramid.httpexceptions import HTTPGatewayTimeout
-import requests.exceptions
+from requests import ConnectionError
+from requests import HTTPError
+from requests import ReadTimeout
+from requests import Response
+from requests import TooManyRedirects
```

Comment:
Real exception objects imported directly to avoid name conflict with `requests` fixture below